clover_web.rb

```diff
@@ -156,7 +156,12 @@ def assets
       Validation.validate_account_name(account[:name])
     end
     after_create_account do
-      Account[account_id].create_project_with_default_policy("Default")
+      account = Account[account_id]
+      account.create_project_with_default_policy("Default")
+      ProjectInvitation.where(email: account.email).each do |inv|
+        account.associate_with_project(inv.project)
```

Comment:
I'm not sure about auto-joining to invited projects. What if I'm invited to project that I don't want to join? Signing up would make me join those projects. We also don't have a way to exit from a project.

That said, I'm also not entirely sure about what should be the ideal experience here. Do you know how does other providers handle this case?

Reply:
We checked a few other providers with Burak offline. It looks like they use auto-join. So we decided to keep it that way for now.